test/modules/XMLHttpRequest/intercept/XMLHttpRequest.test.ts

```diff
@@ -347,3 +349,29 @@ test('sets "credentials" to "omit" on isomorphic request when "withCredentials"
   const [request] = resolver.mock.calls[0]
   expect(request.credentials).toBe('omit')
 })
+
+test('responds with an ArrayBuffer when "responseType" equals "arraybuffer"', async () => {
```

Comment:
This test is broken and should be fixed (it fails [as mentioned here](https://github.com/mswjs/interceptors/issues/309#issuecomment-1328052958)).

Reply:
I've fixed this one, see my comment below.